lib/filterx/filterx-variable.h

```diff
@@ -71,17 +70,15 @@ typedef struct _FilterXVariable
    *
    * declared -- this variable is declared (e.g. retained for the entire input pipeline)
    */
-  guint32 assigned:1,
-          variable_type:2,
```

Comment:
The number of values in `FilterXVariableType` is not checked against this `:2` bitfield. Can we add a static assert to make sure it breaks at compile time when one adds 2 more fields?

Reply:
fixed